sympy/printing/latex.py

```diff
@@ -280,6 +280,19 @@ def _print_Add(self, expr, order=None):
 
         return tex
 
+    def _print_Cycle(self, expr):
+        from sympy.combinatorics.permutations import Permutation, Cycle
+        if str(expr) == '()':
+            return r"\left( \right)"
+        expr_perm = Permutation(expr).cyclic_form
+        term_tex = ''
+        for i in expr_perm:
+            term_tex += str(i).replace(',', r"\thickmuskip")
```

Comment:
This should be ``\;`` to actually create the space. (My use of the name of space may have been misleading, I'm sorry.)